spec/models/purchase_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe Purchase do
+
+  let (:purchase) { Purchase.create(email: "me@example.com", item_id: 2) }
```

Comment:
This item is only used in one of your contexts. Would it make sense to move it in there?